test-network/install-fabric.sh

```diff
@@ -0,0 +1,313 @@
+#!/bin/bash
+#
+# Copyright IBM Corp. All Rights Reserved.
+#
+# SPDX-License-Identifier: Apache-2.0
+#
+
+# A modified version of the Fabric bootstrap script
+# Use positional arguments to select componenets to install
+#
+# Has exactly the same functional power of bootstrap.sh
+
+### START OF CODE GENERATED BY Argbash v2.9.0  ###
+# Argbash is a bash code generator used to get arguments parsing right.
+# Argbash is FREE SOFTWARE, see https://argbash.io for more info
+# Generated online by https://argbash.io/generate
+
+# Default values
+_positionals=()
+_arg_comp=('' )
+
+# if version not passed in, default to latest released version
+# if ca version not passed in, default to latest released version
+_arg_fabric_version="2.4.0"
```

Comment:
2.4.1 ?   2.4 ?   Is there a way to align this somehow with the script returned in the bit.ly link?   Not just the revision, but in how/where it comes from?

I don't like that we need to "install" fabric before using it.  This is way out of context of this PR, but I will take the opportunity to rant here a little bit.   Users should NOT need to install binaries locally to run a container-based application.   Running `peer` commands locally to administer a network .... maybe.  But for casual consumption and sample usage - no.

Reply:
There's a PR on fabric (https://github.com/hyperledger/fabric/pull/3208) where this install script really lives.
WIll remove it from this PR..

(and yes I agree with your last point)